presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.275
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Improve the memory limit error message with allocationTag.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`. The prototype of the function is: trimmed_mean(tdigest: TDIGEST, lower_quantile: DOUBLE, upper_quantile: DOUBLE) -> DOUBLE.
+* Add a router in front of presto clusters. The router is deployed as a standalone service with a presto-styled UI.
```

Comment:
This probably can go to the router changes section. "In front of": I may use the words "to route queries to .. "

Reply:
Normally, when we add a brand new infra component like this, especially for the first time, do we announce it in General Changes?